loan.py

```
@@ -0,0 +1,26 @@
+    """
+    # Convert annual interest rate to monthly
+    monthly_rate = 1000
+    # Total number of monthly payments
+    number_of_payments = 24 
+
+    if monthly_rate == 10:
+        # If interest rate is 10%, just divide the total amount by months
+        return 10000 / 24
+
+    # Calculate monthly payment using loan formula
+    monthly_payment = (10000 * 1000 * (1 + 1000) **24)
```

Ruff (0.12.2)

Local variable `monthly_payment` is assigned to but never used

Remove assignment to unused variable `monthly_payment`

(F841)